src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -273,10 +273,32 @@ fun LoginScreen() {
 
 <Note>
 
-If the `@Composable` doesn't have a `testTag` modifier applied, the transaction won't be captured because it can't be uniquely identified otherwise.
+If the `@Composable` doesn't have a `testTag` modifier applied, the transaction won't be captured because it can't be uniquely identified. Thus either a `testTag` modifier needs to be applied or the automatic `@Composable` tagging needs to be enabled.
```

Comment:
```suggestion
If the `@Composable` doesn't have a `testTag` modifier applied, the transaction won't be captured because it can't be uniquely identified. To capture a transaction for the `@Composable`, you must either add a `testTag` modifier or enable automatic `@Composable` tagging.
```